File: mobile/socket/messages/server/StockItemChangeMessage.ts
export const STOCK_ITEM_CHANGE = "STOCK_ITEM_CHANGE";

export interface StockItemChangePayload {
  id: string;
  price: number,
  timestamp: number,
  changePercent: number,
  change: number,
}

export interface StockItemChangeMessage {
  type:typeof STOCK_ITEM_CHANGE;
  payload: StockItemChangePayload;
}

File: stocks-service/stocksserver/messages/server/StockItemChangeMessage.py
STOCK_ITEM_CHANGE = "STOCK_ITEM_CHANGE"

class StockItemChangePayload:
    def __init__(self, id,price,timestamp,changePercent,change):
        self.id, self.price, self.timestamp,self.changePercent, self.change = id,price,timestamp,changePercent,change
    
class StockItemChangeMessage:
    def __init__(self, payload):
        self.type, self.payload = STOCK_ITEM_CHANGE, payload

